react/features/chat/components/AbstractMessageContainer.js

```diff
@@ -32,15 +37,25 @@ export default class AbstractMessageContainer<P: Props> extends PureComponent<P>
         const groups = [];
         let currentGrouping = [];
         let currentGroupParticipantId;
+        const localParticipant = getLocalParticipant(APP.store.getState());
 
         for (let i = 0; i < messagesCount; i++) {
             const message = this.props.messages[i];
+            const splitedMessage = message.message.split('_');
```

Comment:
i think this need to be replaced with regex match, because not to bug with some user typed text. Also please use some other format - i.e. `[ak_address] ` because for mobile app this will still be visible for users, better be readable (else now it will look like `ak_somelong82013490qewr0qwer_hello, how are you`) and it will also better split on multiple lines